automatic/jenkins/update.ps1

```diff
@@ -0,0 +1,42 @@
+#requires -Modules au, Wormies-AU-Helpers
+param(
+    # The URL to search for information on the latest release
+    $ReleasePage = "https://www.jenkins.io/download/",
+
+    # The regex used to search for the latest version on the ReleasePage
+    $ReleaseRegex = "Download Jenkins`n(?<Version>[\d\.]+)`nLTS",
+
+    # The base of the download URL
+    $DownloadURL = "https://mirrors.jenkins-ci.org/windows-stable/"
+)
+
+function global:au_BeforeUpdate {
+    Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+    @{
+        ".\legal\VERIFICATION.txt" = @{
+            "<PACKAGE VERSION>" = $Latest.Version
+            "<CHECKSUM>"        = $Latest.Checksum64
+            "<CHECKSUMTYPE>"    = $Latest.ChecksumType64
```

Comment:
My understanding is that AU will push back the changes to this repository, made in the updated package, `<PACKAGE VERSION` etc. will not be there after the first push.

I'll let @AdmiringWorm confirm that.

Reply:
I was actually just thinking about this looking at the nexus one. I'll adjust it to match and replace the line, as others do. May have to remove the version replacement, though.